subcommands/el2g/devices.go

```diff
@@ -50,6 +50,14 @@ fioctl el2g devices add --production 935414457472 0x04005001eee3ba1ee96e60047e57
 	}
 	add.Flags().BoolVarP(&production, "production", "", false, "A production device")
 	devicesCmd.AddCommand(add)
+	del := &cobra.Command{
+		Use:   "delete <NC12 product-id> <device-id>",
+		Short: "Remove a device from EdgeLock2GO",
```

Comment:
IMHO as you have the `grant access to el2go` in the add command; here `revoke access from el2go` seems complementary.

Reply:
https://github.com/foundriesio/fioctl/pull/179/commits/9193a8ffda13fab7d2564178dd9f0220a9d81f49